src/interface_adapter/logged_in/LoggedInViewModel.java

```diff
@@ -7,6 +7,12 @@
 public class LoggedInViewModel extends ViewModel {
 
     private LoggedInState state = new LoggedInState();
+    public static final String LOGOUT_BUTTON_LABEL = "Log out";
+    public static final String RECIPEGETTER_BUTTON_LABLE = "Get Recipe";
+    public static final String INVENTORY_BUTTON_LABEL = "Add/Delete Items in Inventory";
+    public static final String RESTRICTION_BUTTON_LABEL = "Add/Delete Items in Restriction";
+
+
```

Comment:
Maybe consider to declare them as static final rather than public static final. This is because these labels typically do not change, and for the sake of consistency and memory efficiency, it's better to declare them as static constants.

Reply:
we need it to be public